tests/query_test.py

```diff
@@ -305,6 +305,14 @@ def test_time(self):
     time = "1970-01-01T00:00:00.123456789Z"
     assert self._q(query.time(time)) == FaunaTime(time)
 
+    # 'now' refers to the current time.
+    now = self._q(query.time('now'))
+    assert 0 < (datetime.now() - now.to_datetime()).microseconds < 1000000
```

Comment:
There's no need to test core's behavior here, as long as you get a timestamp back.